src/collisions-resolver.ts

```diff
@@ -11,6 +11,7 @@ import { verboseLog } from './logger';
 
 const renamingSupportedSymbols: readonly ts.SymbolFlags[] = [
 	ts.SymbolFlags.Alias,
+	ts.SymbolFlags.FunctionScopedVariable,
 	ts.SymbolFlags.BlockScopedVariable,
```

Comment:
```suggestion
	ts.SymbolFlags.Variable,
```

looks like we can merge them into 1 symbol now